### DISCUSSION TITLE
Bulk orders

### DISCUSSION THREAD

**Discussion by ffapparel** [swellstores/community#195] [Help] [answered]
Hello, we are trying build out a size/qty product matrix for bulk ordering using Horizon theme.  Are there any plans to enable Horizon cart to accept more than one product at a time?

**Comment by sags95** [comment, marked as answer]
@ffapparel This is not currently on the roadmap, but this is supported within swell-js https://developers.swell.is/frontend-api/carts#update-multiple-items. So the implementation would mainly be building a UI around this function.

**Reply by ffapparel** [reply]
Thank you for your comment.  Does this mean we CAN use the Horizon theme but just need UI fix?  or do we need to build a new custom theme completely?  We have two developers that have attempted it within Horizon but hit roadblocks.

**Reply by logeshswell** [reply]
> Does this mean we CAN use the Horizon theme but just need UI fix?

Yes, it requires a UI fix and development efforts to build on top of Horizon using the [update-multiple-items](https://developers.swell.is/frontend-api/carts#update-multiple-items) method in swell-js.
